### DISCUSSION TITLE
Important information not available on rxjs.dev

### DISCUSSION THREAD

**Discussion by ineffective** [ReactiveX/rxjs#7085] [Report issues other than bug]
As someone who has no background in JS, TS or Angular I struggled to find answer to [my question on StackOverflow](https://stackoverflow.com/questions/74033486/why-rxjs-mergemap-turns-array-into-separate-objects). After receiving some help I looked deeper and found out that [this document](https://reactivex.io/rxjs/class/es6/MiscJSDoc.js~ObservableInputDoc.html) answers all my questions, while [this page](https://rxjs.dev/api/index/type-alias/ObservableInput) describing the same thing, is absolutely useless. 
Moreover I don't quite get what is the relation between rxjs.dev and reactivex.io sites. Is one just a new version of the other? Are they for different implementations?

I have piece of Angular in my project and I have to work on it for the time being. It is full of rxjs chains/pipes, and I frequently struggle to find an obvious answer to my questions - of course now, knowing that reactivex.io also contains very useful stuff I will check it, but it would be nice if all these things could be found on one.

**Comment by jakovljevic-mladen** [comment]
Hi @ineffective, I wrote about this issue [here](https://github.com/ReactiveX/rxjs/discussions/6954#discussioncomment-3195739), please take a look. I will be working on adding more documentation to the `ObservableInput` and that is something I plan to work on after I finish some of the work related to the last set of V7 features.

> Moreover I don't quite get what is the relation between rxjs.dev and reactivex.io sites.

The official RxJS docs site is https://rxjs.dev.

I joined RxJS team this year and I started contributing to RxJS about 3 years ago. The official site is being generated from the `/docs_app` folder which was [added more than 4 years ago](https://github.com/ReactiveX/rxjs/pull/3763) which is way older than my first contributions. So, I can't really tell how reactivex.io is related to the RxJS. It looks like it's an old and deprecated docs site, but I don't know for sure.

**Reply by ineffective** [reply]
I understand. Thank you for this information. Is there anything I can do to help?

**Reply by jakovljevic-mladen** [reply]
Docs PRs are always welcome ☺️
